frontend/src/Editor/Inspector/EventSelector.jsx

```diff
@@ -115,10 +126,15 @@ export const EventSelector = ({
               {definition.actionId === 'go-to-app' && (
                 <div className="p-1">
                   <label className="form-label mt-1">App</label>
-                  <CodeHinter
-                    currentState={currentState}
-                    initialValue={definition.options.appId}
-                    onChange={(value) => eventOptionUpdated(param, 'appId', value, extraData)}
+                  <SelectSearch
+                    options={getAllApps()}
+                    search={true}
+                    value={definition.options.appId}
+                    onChange={(value) => {
+                      eventOptionUpdated(param, 'appId', value, extraData);
```

Comment:
It should be `slug` instead of `appId` right? I think I made that mistake while building the go to app action. Can you please clean that up in this PR?

Reply:
Done.